pkg/reconciler/trigger/trigger.go

```diff
@@ -98,6 +98,11 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, t *brokerv1beta1.Trigger
 		return err
 	}
 
+	if apierrs.IsNotFound(err) {
+		logging.FromContext(ctx).Error(fmt.Sprintf("Trigger %s/%s has no broker %q", t.Namespace, t.Name, t.Spec.Broker))
```

Comment:
`Errorf`?

Reply:
I think in this case we just want to log an error message rather than generate an event